en/integrations/plugin-packs/procedures/network-switchs-symbol-wing-snmp.md

```diff
@@ -0,0 +1,179 @@
+---
+id: network-switchs-symbol-wing-snmp
+title: Symbol WiNG switch SNMP
+---
+
+## Pack Assets
+
+### Templates
+
+The Centreon Plugin Pack Symbol WiNG switch SNMP brings 1 host template:
+* Net-Symbol-Wing-SNMP-custom
+
+It brings the following Service Templates:
+
+| Service Alias        | Service Template                          | Default | Discovery |
+|:---------------------|:------------------------------------------|:--------|:----------|
+| Systems              | Net-Symbol-Wing-Systems-SNMP              | X       |           |
+| Traffic-Generic-Id   | Net-Symbol-Wing-Traffic-Generic-Id-SNMP   |         |           |
+| Traffic-Generic-Name | Net-Symbol-Wing-Traffic-Generic-Name-SNMP |         | X         |
+| Traffic-Global       | Net-Symbol-Wing-Traffic-Global-SNMP       |         |           |
+| Packet-Errors        | Net-Symbol-wing-Packet-Errors-Global-SNMP |         | X         |
+
+### Discovery rules
+
+| Rule name                               | Description |
+|:----------------------------------------|:------------|
+| Net-Symbol-Wing-SNMP-Traffic-Name       |             |
+| Net-Symbol-Wing-SNMP-Packet-Errors-Name |             |
+
+### Collected metrics & status
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Systems-->
+
+* Global
+
+| Metric name         | Description       | Unit  |
+|:--------------------|:------------------|:------|
+| devices.total.count | total devices: %s | count |
+
+* Per *cpu*
+
+| Metric name                    | Description   | Unit |
+|:-------------------------------|:--------------|:-----|
+| cpu.utilization.1m.percentage  | %.2f %% (1m)  | %    |
+| cpu.utilization.5m.percentage  | %.2f %% (5m)  | %    |
+| cpu.utilization.15m.percentage | %.2f %% (15m) | %    |
```

Comment:
A description is needed, the script is not that smart 😛

Reply:
done